src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+.body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --blue-color: #00acdc;
+}
+
+.header {
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  padding: 0 50px;
+}
+
+.logo__img {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-weight: 500;
+  display: inline;
```

Comment:
```suggestion
```
looks like redundant . it is default value. better to set some other not inlive display (inline-block, flex, block)